<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProfileFromPointsDialogBase</class>
 <widget class="QDialog" name="ProfileFromPointsDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>881</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ProfileFromPoints</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="layoutPlot">
     <property name="sizeConstraint">
      <enum>QLayout::SetNoConstraint</enum>
     </property>
     <item row="1" column="0">
      <widget class="QWidget" name="toolbarWidget" native="true"/>
     </item>
     <item row="0" column="0">
      <widget class="QWidget" name="canvasWidget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <zorder>groupBox</zorder>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Options</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QFormLayout" name="formLayout">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::FieldsStayAtSizeHint</enum>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Point layer</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="uPointLayer">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Elevation field</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="uZfield">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Point ID</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QComboBox" name="uPointID"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>No Data Value</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="lineEditNoData">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QCheckBox" name="uSelectedPoints">
          <property name="text">
           <string>Use only selected points</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="uPointIDlabels">
          <property name="text">
           <string>Label plot with Point ID</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="uNoHeader">
          <property name="text">
           <string>Copy data w/o headers or Point ID (for HEC-RAS)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="uCopytoClip">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Copy Station and Elevation to Clipboard</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QTabWidget" name="utabWidget">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="utabCreateLine">
           <attribute name="title">
            <string>Create profile line</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>This option will create a memory layer called profileFromPointsLine by connecting points using the order field and sort order.</string>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>Order field</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="uOrderField">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>200</width>
                  <height>0</height>
                 </size>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>Sort order</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="uOrder">
                <item>
                 <property name="text">
                  <string>Ascending</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Descending</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="utabExistingLine">
           <attribute name="title">
            <string>Use existing line</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>This option will station the points using the line layer.</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QLabel" name="label_5">
                <property name="text">
                 <string>Line layer</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="uLineLayer">
                <property name="minimumSize">
                 <size>
                  <width>200</width>
                  <height>0</height>
                 </size>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <item>
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>Buffer distance from line (line layer units)</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="uBuffer">
                <property name="autoFillBackground">
                 <bool>true</bool>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QCheckBox" name="uSelectedLine">
              <property name="text">
               <string>Use selected line only</string>
              </property>
              <property name="checked">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QDialogButtonBox" name="buttonBox">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="standardButtons">
           <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
